### DISCUSSION TITLE
Aleph's document?

### DISCUSSION THREAD

**Discussion by dhuyartmaker** [alephjs/aleph.js#480] [Q&A]
Where can i read document of Aleph?
I try https://alephjs.org/docs but they are one similar page
<img width="1108" alt="Screen Shot 2022-05-27 at 4 06 42 PM" src="https://user-images.githubusercontent.com/102408799/170668625-5d553e66-4ab5-44b2-91e9-67228323730c.png">
.

**Comment by ije** [comment]
sorry, i am still working on it , it's not ready yet
